templates/blocks/site_logo.twig

```diff
@@ -4,7 +4,8 @@
 	{% if logo == 'greenpeace' %}
 		<img src="{{ data_nav_bar.images }}gp-logo-{{ logo_color }}.svg" alt="Greenpeace">
 	{% else %}
-		<img src="{{ data_nav_bar.images }}{{ logo }}/logo-{{ logo_color }}.png" alt="Greenpeace">
+		{% set color = logo_color == 'green' ? 'light' : logo_color %}
```

Comment:
This is caused by another issue, it was added as a separate commit so we can revert it when the issue is solved.

Reply:
Details in the follow up ticket: https://jira.greenpeace.org/browse/PLANET-4901